code/modules/antagonists/changeling/powers/biodegrade.dm

```diff
@@ -67,9 +68,19 @@
 		user.SetWeakened(0)
 		playsound(user.loc, 'sound/weapons/sear.ogg', 50, TRUE)
 		used = TRUE
+
 	if(used)
 		SSblackbox.record_feedback("nested tally", "changeling_powers", 1, list("[name]"))
-	return TRUE
+		return TRUE
+
+	if(!current_hand)
+		if(!add_hand_spell(user))
+			return FALSE
+		to_chat(user, "<span class='warning'>We prepare hand to vomit acid!</span>")
+		return TRUE
+	else
+		remove_hand_spell(user, TRUE)
+		return FALSE
```

Comment:
no need for the else, just bring it up a level

Reply:
updated